lib/datadog/tracing/contrib/elasticsearch/patcher.rb

```diff
@@ -77,6 +77,13 @@ def perform_request(*args)
                     host = connection.host[:host] if connection
                     port = connection.host[:port] if connection
 
+                    if datadog_configuration[:peer_service]
+                      span.set_tag(
+                        Tracing::Metadata::Ext::TAG_PEER_SERVICE,
+                        datadog_configuration[:peer_service]
+                      )
+                    end
```

Comment:
Is this something that could be encapsulated inside `SpanAttributeSchema`? 

Also, not necessary for this PR but I believe the name `SpanAttributeSchema` no longer reflect the core responsibility of that module. Maybe `PeerServiceTagger`?

Reply:
After discussing, we decided not to move forward with these suggestions. We want to avoid sending the span and configuration block to `SpanAttributeSchema` more than necessary, and regardless of the encapsulation, it still needs to be called every integration so there seems to be no need to change it.

While it may not reflect changes at the moment, there will be many changes in the future, fully showing the reason for calling it `SpanAttributeSchema`